java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -34,6 +34,25 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> get(String key);
 
+    /**
+     * Gets the value associated with the given <code>key</code>, or <code>null</code> if no such
+     * value exists.
+     *
+     * @see <a href="https://redis.io/commands/get/">redis.io</a> for details.
+     * @param key The <code>key</code> to retrieve from the database.
+     * @return Response from Redis. If <code>key</code> exists, returns the <code>value</code> of
+     *     <code>key</code> as a <code>String</code>. Otherwise, return <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * byte[] value = client.get("key").get();
+     * assert Arrays.equals(value, "value".getBytes());
+     *
+     * String value = client.get("non_existing_key").get();
+     * assert value.equals(null);
+     * }</pre>
+     */
+    CompletableFuture<byte[]> get(byte[] key);
```

Comment:
The key name could be a simple string, but value would be anything.
`String get(String)` fails in such case, using `byte[]> get(byte[])` isn't trivial and user-friendly.